bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -141,60 +177,87 @@ final class BagsManager(executorService: ExecutorService) {
 
         val key = obs.getNodeKey
         state += key
+        if (bagsStatus(key).isEmpty)
+          setStatus(key, BagsStatus.NewPending.some)
+
         Future {
+          // Wait the delay before running.
+          Thread.sleep(delay)
+
           // If dequeue is false this means that (a) another task scheduled *after* me ended up
           // running before me, so their result is as good as mine would have been and we're done;
           // or (b) we don't care about that program anymore, so we're done.
           if (dequeue(key, obs.getProgramID)) {
```

Comment:
I was trying to find a way to highlight them and flag you, but you beat me to it 😄